### PR TITLE
Use SecretsManager for RDS credentials

### PR DESCRIPTION
**Pull request by jonnyshaw89**
This PR removes the `DB_CONNECTION_URI` environment variable in favour of the `DB_SECRET_ID` environment variable.

This allows the app to load the database secrets on startup so the api does not need to be passed any secrets.

Running this PR also tests https://github.com/ONSdigital/eq-terraform/pull/155

### CODE REVIEW

**Review by tgandrews — APPROVED** [review]
LGTM although I can't run to prove it.

File: developer_defaults.tf [hunk 1]
```diff
@@ -57,7 +57,7 @@ variable "survey_runner_min_tasks" {
 
 variable "respondent_account_url" {
   description = "The url for the respondent log in"
-  default     = "https://survey.ons.gov.uk/"
+  default     = "https://surveys.ons.gov.uk/"
```
Comment (samiwel) [inline developer_defaults.tf:60]: Why was this changed?

**Review by samiwel — APPROVED** [review]
Looks good, one question.

**Comment by jonnyshaw89** [comment]
Closing as it doesn't make sense with DynamoDB
